ncm-metaconfig/src/main/pan/components/metaconfig/schema.pan

```diff
@@ -15,7 +15,6 @@ type ${project.artifactId}_config =  {
      'mode' : long = 0644
      'owner' : string = 'root'
      'group' : string = 'root'
-     'daemon' ? string[]
```

Comment:
I was under the impression from the workshop that it is already deprecated (hence the issue) and  why I'm stripping it out now. If i misunderstood then we should push it back a release or two, but I really don't want anyone else to start the deprecated structure in the mean time.